DSESwift/DataDeletionRequest.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  RLMDataDeletionRequest.swift
+//  SensePlatform
+//
+//  Created by Fei on 15/10/15.
+//
+//
+
+import Foundation
+import RealmSwift
+
+class DataDeletionRequest: Object {
+    dynamic var uuid = ""
+    dynamic var userId = ""
+    dynamic var sensorName = ""
+    dynamic var sourceName = ""
+    dynamic var startDate: Double = -1.0  // -1.0 as null by default, if not null, change from default
```

Comment:
@orhan89 Realm does not support nil for Double on the version we use. Right? @excaliburnan
I have also heard that the latest release of Realm supports nil.